docs/src/main/sphinx/release/release-397.md

```diff
@@ -0,0 +1,46 @@
+# Release 397 (21 Sep 2022)
+
+## General
+
+* Fix incorrect parsing of invalid values in cast from `varchar` to `timestamp`. ({issue}`14164`)
+* Fix potential incorrect results for queries where all data scanned from tables
+  was processed in the leaf stage and was not passed downstream. ({issue}`14168`)
+* Fix `EXPLAIN (TYPE IO)` query failure for certain queries with redundant table
+  scans. ({issue}`10398`)
+
+## JDBC driver
+
+* Fix potential memory leak when cancelling statements. ({issue}`14176`)
```

Comment:
drop "potential" maybe?

Reply:
I think we've used the word "potential" a lot for things that occur sporadically, under specific circumstances, or both. The memory leak doesn't happen every time you cancel a statement, so we need to hedge it.

Same generally applies for the comment you made above.